qos/evm/qos.go

```diff
@@ -14,9 +14,18 @@ func NewQoSInstance(logger polylog.Logger, config ServiceConfig) *QoS {
 	)
 
 	serviceState := &ServiceState{
-		logger:              logger,
-		chainID:             evmChainID,
-		archivalCheckConfig: config.GetEVMArchivalCheckConfig(),
+		logger:  logger,
+		chainID: evmChainID,
+	}
+
+	if archivalCheckConfig, enabled := config.GetEVMArchivalCheckConfig(); enabled {
```

Comment:
Not a blocker: not sure why running archival checks is optional: 
IMO, if not configured we should:
1. Use a set of default values.
2. Log a warning.

Reply:
My assumption has been that we don't want to enforce that ALL services much be archival. For example, in the future we may want to allow running pruned EVM nodes in some services.

I know currently in Morse all Grove's chains are archival (F chains) but I don't think that this assumption should be baked into the code of PATH.

Tagging @Olshansk for input here as well.